data/i18n/en/en.toml

```diff
@@ -279,6 +279,15 @@ other = "Thanks for the feedback. If you have a specific, answerable question ab
 [layouts_docs_search_fetching]
 other = "Fetching results..."
 
+[legacy_repos_message]
+other = """Kubernetes has [new package repositories hosted at `pkgs.k8s.io`](/blog/2023/08/15/pkgs-k8s-io-introduction/)
+starting from August 2023. The legacy package repositories (`apt.kubernetes.io` and `yum.kubernetes.io`) have been
```

Comment:
Do we want to mention that they're identical with `apt.k8s.io` and `yum.k8s.io` to avoid confusion?

Reply:
No, I'd try to avoid making this assumption. In fact, they are not identical:

- Revisions are different
- Packages are different (we built new packages for pkgs.k8s.io, so hashes and signatures are different)
- We had that bug in the kubeadm package where we used incorrect `sysconfig` file (it's now fixed)

The message is already long enough, and there's a link to the blog post that clarifies everything.